hazelcast/src/main/java/com/hazelcast/nio/tcp/handlermigration/IOBalancer.java

```diff
@@ -178,26 +179,4 @@ private void tryMigrate(LoadImbalance loadImbalance) {
         }
         handler.migrate(destinationSelector);
     }
-
-    private final class IOBalancerThread extends Thread {
```

Comment:
I'm fine with leaving since it a flavor thing. But the thread is small and pokes in the internal of that class. The thread is tied to the implementation, so I make it internal. Unless it becomes big, then I rip it apart.